src/API/Management/Tickets.php

```diff
@@ -80,21 +82,25 @@ public function createPasswordChangeTicket(
         $connection_id = null,
         $ttl = null,
         $client_id = null,
-        $organization_id = null
+        $organization_id = null,
```

Comment:
> adding more parameters to existing public functions, wouldn't this be a breaking change? assuming a developer could be invoking the function without using named parameters.
> 
> e.g.
> 
> ```
> // call fails because is missing the 3 params
> createPasswordChangeTicket(userId, newPass, resultUrl, connectionId, ttl, clientId) 
> ```

Hey @lbalmaceda 👋 Not in this particular case, as the parameters were added to the end of the method call and all the parameters are nullable, there's no potential for breakage.